.vale.ini

```diff
@@ -0,0 +1,13 @@
+StylesPath = .github/styles
+MinAlertLevel = suggestion
+Vocab = Keptn
+
+[*.{md,mdx}]
+BasedOnStyles = Vale, Google
+
+Vale.Spelling = warning
```

Comment:
What does this do? I couldn't find the documentation around this.

Reply:
Vale comes with built-in styles. One of its features is Spell-Check and this configuration basically says whenever you encounter a special mistake, show it as a warning. I hope it clears it up.